src/server_manager/web_app/app.ts

```diff
@@ -121,6 +122,23 @@ async function showHelpBubblesOnce(serverView: Polymer) {
   }
 }
 
+// Displays a survey dialog with title `surveyTitle` and a link to `surveyLink`.
+// Does not display the survey with `surveyId` if it has already been shown to
+// the user or if the current date is after `displayBefore`.
```

Comment:
It seems we can simplify. Do we really need the displayBefore? If the feature is not there, you won't trigger the survey.  If it's there permanently, it means we made a code change, so we might as well remove the survey. I also imagine we may want to keep collecting some survey results.

Reply:
@cjhenck, could you please add context to this requirement?